### DISCUSSION TITLE
[Q&A] 上传文件、调用DALLE3、askpdf插件使用咨询

### DISCUSSION THREAD

**Discussion by Payne-Wang** [lobehub/lobehub#795] [General Question]
### 🧐 问题描述 | Proposed Solution

大佬好，在使用过程中有几个问题想请教: 

ps:我是使用vercle部署的

1.是否有上传文件的功能，我有一个类似于plus上传东西文件的中转的api，想用用看

2.是否支持DALLE3？如何设置？

3.插件askpdf和授粉绘画插件如何使用？

谢谢大佬！

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
> 1.是否有上传文件的功能，我有一个类似于plus上传东西文件的中转的api，想用用看

暂不支持

> 2.是否支持DALLE3？如何设置？

已支持，升级到最新版本就有了

> 3.插件askpdf和授粉绘画插件如何使用？

安装后打开就能用呀

**Reply by Payne-Wang** [reply]
谢谢，请问那个askpdf，我该如何上传我的pdf让他解读呢？打开插件后没有发现上传按钮。我是vercel部署的

**Reply by LBEILC** [reply]
那个插件你可以发pdf的链接给他 如果去他的官网上传pdf会得到一个文件的链接 把那个发给他也可以

**Reply by Payne-Wang** [reply]
感谢，学到了！谢谢

**Comment by jkhcc11** [comment]
> 那个插件你可以发pdf的链接给他 如果去他的官网上传pdf会得到一个文件的链接 把那个发给他也可以

有个问题，如果我不充值官方那个会员计划，只注册账号，然后上次文件，是不是就可以一直用它这个分析插件？
